packages/web3-eth-abi/CHANGELOG.md

```diff
@@ -39,7 +39,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-#### web3-eth-abi
+### Added
 
--   Return `BigInt` instead of `string` when decoding function parameters for large numbers, such as `uint256`.
 -   If an error happens when decoding a value, preserve that exception at `innerError` inside the `AbiError`.
+-   Add basic functionality that is used, by `web3-eth-contract`, when decoding error data according to EIP-838.
+
+### Fixed
+
+-   Return `BigInt` instead of `string` when decoding function parameters for large numbers, such as `uint256`.
```

Comment:
For this and all other changelog entries could you add PR number as: 
```suggestion
-   Return `BigInt` instead of `string` when decoding function parameters for large numbers, such as `uint256`. (#5434)
```